app/controllers/users_controller.rb

```diff
@@ -35,6 +35,7 @@ def show
 
   # GET /users/1/edit
   def edit
+    @page_title = ts("Edit Profile | #{ArchiveConfig.APP_NAME}") 
```

Comment:
This will definitely produce the desired result, but it's not _quite_ the best way to go about it. 

As you noticed, `@page_title` overwrites the entire title, so you have to include ` | #{ArchiveConfig.APP_NAME}`. 

But we also have `@page_subtitle`, which just sets the text before the vertical bar and app name:

https://github.com/otwcode/otwarchive/blob/9ce7f82a0e0eaca675a7899222b805eed421b57c/app/views/layouts/application.html.erb#L21-L32

It would also be a good idea to set this up for eventual interface translation, like we've done with this page title:

https://github.com/otwcode/otwarchive/blob/9ce7f82a0e0eaca675a7899222b805eed421b57c/app/controllers/admin/blacklisted_emails_controller.rb#L10
https://github.com/otwcode/otwarchive/blob/9ce7f82a0e0eaca675a7899222b805eed421b57c/config/locales/views/en.yml#L188-L189

What's happening in the controller there is we're taking advantage of [Rails i18n's lazy lookup](https://guides.rubyonrails.org/i18n.html#lazy-lookup) to tell the app to check our locale files for the value located at:

```
admin:
  blacklisted_emails:
    index:
      browser_title:
``` 

So in this file, you'd write `@page_title = t(".browser_title")` to tell Rails to to look up the value located at:
```
  users: # The controller
    edit: # The action
      browser_title: # The key we've specified
```

That value doesn't currently exist, of course, so you'd need to add it under [users](https://github.com/otwcode/otwarchive/blob/9ce7f82a0e0eaca675a7899222b805eed421b57c/config/locales/views/en.yml#L716), e.g.:

```
  users:
    edit:
      browser_title: Edit Profile
    change_username:
      account_faq: Account FAQ
      caution: Please use this feature with caution.
```

I think you mentioned you were new to Rails, so here's the [Rails i18n documentation](https://guides.rubyonrails.org/i18n.html) as well as our [i18n Standards](https://github.com/otwcode/otwarchive/wiki/Internationalization-%28i18n%29-Standards), which also explains some of the automated tasks we use for i18n. Those automated tasks will alphabetize and reformat any changes you make to config/locales/views/en.yml, so it's a good idea to run them before committing. 

Lastly -- and I'm sorry this is so much! -- it would be a good idea to add an automated test for this in [features/other_a/profile_edit.feature](https://github.com/otwcode/otwarchive/blob/master/features/other_a/profile_edit.feature#L28-L30).

That file is set up in a way that makes adding the test a little awkward, so it's probably best to add `Then I should see the page title "Edit Profile"` above the existing the first line (`When I fill in the details of my profile`) of the ["Add details" scenario](https://github.com/otwcode/otwarchive/blob/9ce7f82a0e0eaca675a7899222b805eed421b57c/features/other_a/profile_edit.feature#L14C1-L18C37).